package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "supertokens-react-native",
-  "version": "4.0.8",
+  "version": "4.1.0",
```

Comment:
missing change below as well.. there is "version": "4.0.8", right below too

Reply:
Done